### ISSUE TITLE
Uncaught SyntaxError: Unexpected token '<'

### ISSUE THREAD

**Issue by Esugenos** [preactjs/wmr#339]
Hey,

I'm new to wmr, it looks very nice. But I recently encountered a bug (at least, it looks like a bug to me) 

## Bug

Parsing error 
`Uncaught SyntaxError: Unexpected token '<'` 💩

Lapidary writing style seems to break the parsing step:

```<Route path="/home"><Home/></Route>``` is breaking it

```<Route path="/home"> <Home/> </Route>``` isn't

(that's the `public/components/App.tsx` 9th line of code)

## Reproduction

Clone [the repo](https://github.com/Esugenos/wmr-app/tree/main)

`npm i`

`npm run start`

Open `http://localhost:8080`

Check the console, the error is there

___
___
___
Bonus: 

`GET http://localhost:8080/assets/style.module.7b16dcc3.css net::ERR_ABORTED 404 (Not Found)` when `npm run build` `npm run serve`

**Comment by marvinhagemeister** [comment]
Looks like the current jsx parser stumbles on `<Home/>`. If I add a space there it compiles `<Home />`.

**Comment by marvinhagemeister** [comment]
Here is a simple test case that reproduces the issue:

```jsx
export function Foo() {
	// prettier-ignore
	return <Bar><Bob/></Bar>;
}
```

**Comment by marvinhagemeister** [comment]
FYI: We've just published wmr 1.3.0 which includes the fix for this issue 👍

**Comment by Esugenos** [comment]
thanks, awesome 🚀🚀
